controllers/release/cache.go

```diff
@@ -45,7 +45,7 @@ func SetupReleasePlanAdmissionCache(mgr ctrl.Manager) error {
 		"spec.origin.namespace", releasePlanAdmissionIndexFunc)
 }
 
-// SetupSnapshotEnvironmentBindingCache adds a new index field to be able to search SnapshotEnvironmentBinding by environment.
+// SetupSnapshotEnvironmentBindingCache adds a new index field to be able to search SnapshotEnvironmentBindings by environment.
```

Comment:
ApplicationSnapshotEnvironmentBinding?

Reply:
The resource is getting renamed (Application prefix is getting dropped) so I renamed it already where I could to reduce the size of the patch once GitOps get it renamed.

Reply:
got it. tx